filebeat/crawler/crawler.go

```diff
@@ -7,16 +7,18 @@ import (
 	"github.com/elastic/beats/filebeat/channel"
 	"github.com/elastic/beats/filebeat/fileset"
 	"github.com/elastic/beats/filebeat/input/file"
-	"github.com/elastic/beats/filebeat/prospector"
+	pros "github.com/elastic/beats/filebeat/prospector/prospectors"
 	"github.com/elastic/beats/filebeat/registrar"
 	"github.com/elastic/beats/libbeat/cfgfile"
 	"github.com/elastic/beats/libbeat/common"
 	"github.com/elastic/beats/libbeat/common/cfgwarn"
 	"github.com/elastic/beats/libbeat/logp"
+
+	_ "github.com/elastic/beats/filebeat/prospector"
```

Comment:
Here is for example the script that generates the imports for metricbeat: https://github.com/elastic/beats/blob/master/metricbeat/scripts/generate_imports.py